packages/ngtools/webpack/src/virtual_file_system_decorator.ts

```diff
@@ -26,12 +26,14 @@ export class VirtualFileSystemDecorator implements InputFileSystem {
   }
 
   stat(path: string, callback: (err: Error, stats: Stats) => void): void {
-    try {
+    const result = this._webpackCompilerHost.stat(path);
```

Comment:
`stat` can throw which would stop the callback from running.

Also, the webpack input file system directly uses the node callback which provides the error directly in the callback.  This error could be for any reason and not just that the file doesn't exist.

Reply:
I haven't seen a throw myself, when the fail doesn't exists stats returns `null`, Seeing the code https://github.com/angular/angular-cli/blob/a53cf44e1db15f7e0680539fa9857506072e5104/packages/ngtools/webpack/src/compiler_host.ts#L190-L20. I looks like it is handled
 
The error itself origins from `globby`: https://github.com/sindresorhus/globby/blob/master/index.js#L14

Reply:
So the real issue appears to be here then: https://github.com/webpack-contrib/copy-webpack-plugin/blob/1f9272945957077a623683e1c4d959d477e1555c/src/preProcessPattern.js#L76

That conditional should have the same checks as the catch block.